src/raftstore/store/peer.rs

```diff
@@ -2263,7 +2268,7 @@ impl Peer {
         msg.set_to(self.leader_id());
         msg.set_msg_type(eraftpb::MessageType::MsgTransferLeader);
         msg.set_index(self.get_store().applied_index());
-        msg.set_term(self.term());
+        msg.set_log_term(self.term());
```

Comment:
For followers `log term` may be not equal to `term`. Is it ok?

Reply:
It's checked at L2227.